src/util/checkSpecialCharacters.ts

```diff
@@ -0,0 +1,43 @@
+import { TestCase } from "@madie/madie-models";
+
+export const specialChars = /[`!@#$%^&*()_\+=\[\]{};':"\\|,.<>\/?~]/;
+
+export const EXPORT_ERROR_CHARACTERS_MESSAGE =
+  "Test Cases can not be exported some titles or groups contain special characters.";
+export const SPECIAL_CHARACTERS_ERROR_TITLE =
+  'Test Case Title can not contain special characters: (){}[]<>/|"\':;,.~`!@#$%^&*_+="';
+export const SPECIAL_CHARACTERS_ERROR_SERIES =
+  'Test Case Group can not contain special characters: (){}[]<>/|"\':;,.~`!@#$%^&*_+="';
```

Comment:
`'Test Case Group can not contain special characters: ${specialChars}`

Reply:
@adongare Changed.